uikit/zoomcontrols/src/main/java/com/sygic/ui/view/zoomcontrols/ZoomControlsMenuButton.kt

```diff
@@ -25,7 +25,8 @@ internal class ZoomControlsMenuButton @JvmOverloads constructor(context: Context
     }
 
     fun onMenuAction(open: Boolean) {
-        val drawableRes = if (open) R.drawable.ic_cross else R.drawable.ic_plus_minus
-        setImageDrawableWithFade(getVectorDrawableWithTint(drawableRes, iconColor), ANIMATION_DURATION)
+        val drawable = resources.getDrawable(if (open) R.drawable.ic_cross else R.drawable.ic_plus_minus, null)
```

Comment:
pouzime namiesto `resources.getDrawable(R.drawable.***, null)` `ContextCompat.getDrawable(context, R.drawable.***)`

kvoli tym attributom z temy. podla mna by nas to dostalo za chvilu

Reply:
Souhlas, změněno, díky ;)